PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -117,8 +125,8 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   /// \param collision collision to test against the selection criteria
   /// \param centrality collision centrality variable to be set in this function
   /// \return bitmask with the event selection criteria not satisfied by the collision
-  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll>
-  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality)
+  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename Coll, typename BCs>
+  uint16_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, const BCs& /*bcs*/, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb)
```

Comment:
Also please add `/// \param ccdb ...` in the documentation

Reply:
It should all be done here [d7a1886](https://github.com/AliceO2Group/O2Physics/pull/6533/commits/d7a1886852a4b68448f3d4af61b5e37af57b8fab). Could you please take a look if this is what you had in mind? Thanks!

Reply:
Thanks